src/global-settings.php

```diff
@@ -282,6 +282,53 @@ public function register_global_settings() {
 				)
 			);
 
+			register_setting(
+				'stackable_global_settings',
+				'stackable_font_pair_name',
```

Comment:
I suggest we change this name to something more descriptive to what it is for:
```suggestion
				'stackable_selected_font_pair',
```